tests/EnviroSense.Application.Tests/Authorization/AccessRules/DeviceApiKeyAccessRuleTest.cs

```diff
@@ -73,22 +73,35 @@ private Guid SampleGuid()
         return Guid.Parse("5fe7be6c-4ce6-43ce-94f5-e4f91df55a74");
     }
 
-    private DeviceApiKey SampleDeviceApiKey()
+    private List<ApiKey> SampleDeviceApiKeys()
```

Comment:
```suggestion
    private ApiKey SampleApiKeys()
```

Why do we need list with 2 api keys if we always return first? we can return only used one.